backend/app/service/website_utils.go

```diff
@@ -1037,22 +1036,7 @@ func UpdateSSLConfig(websiteSSL model.WebsiteSSL) error {
 			return buserr.WithErr(constant.ErrSSLApply, err)
 		}
 	}
-	enable, auto, sslID := GetSystemSSL()
-	if enable && sslID == websiteSSL.ID {
-		fileOp := files.NewFileOp()
-		secretDir := path.Join(global.CONF.System.BaseDir, "1panel/secret")
-		if err := fileOp.WriteFile(path.Join(secretDir, "server.crt"), strings.NewReader(websiteSSL.Pem), 0600); err != nil {
-			global.LOG.Errorf("Failed to update the SSL certificate File for 1Panel System domain [%s] , err:%s", websiteSSL.PrimaryDomain, err.Error())
-			return err
-		}
-		if err := fileOp.WriteFile(path.Join(secretDir, "server.key"), strings.NewReader(websiteSSL.PrivateKey), 0600); err != nil {
-			global.LOG.Errorf("Failed to update the SSL certificate for 1Panel System domain [%s] , err:%s", websiteSSL.PrimaryDomain, err.Error())
-			return err
-		}
-		if auto {
-			_, _ = cmd.Exec("systemctl restart 1panel.service")
-		}
-	}
+	reloadSystemSSL(&websiteSSL, nil)
 	return nil
 }
 
```

Comment:
Since the comparison is done between dates before October 2021, I can't find any significant difference between two provided code snippets from now. Both look like correct implementations of an SSL certificate handling utility functions and classes that utilize a database to store configuration settings.

However, in case you're looking specifically for updates on best practices or more recent features, please specify your request accordingly, as this specific snippet lacks context about what exactly needs checking.

For instance, if it would help verify if new APIs have been developed within OpenAI’s model (as mentioned after September 2021), we might want to refer back to their technical documentation or consider using more modern libraries/frameworks for efficiency instead.